docs/data/date-pickers/timezone/BasicValueProp.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export default function BasicValueProp() {
+  const [value, setValue] = React.useState<Dayjs | null>(
+    dayjs.utc('2022-04-17T15:30').tz('America/Chicago'),
+  );
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DemoContainer components={['DateTimePicker']}>
+        <DateTimePicker value={value} onChange={setValue} />
```

Comment:
Right, I'll migrate the demos to the pickers soon, to start really testing this part as well
I'm mostly working with the fields for now.

Reply:
I added the `StoredValue`

For the component being used, I used `DateTimePicker` everywhere to allow user to play with all the views
You you just use `TimePicker` on this one ?

Reply:
for the first demo yes, because I assume user rarely play a lot with them. They just want to see what's the topic of the page.  And here to understand the topic you need to spot that hours are not the same. So a time picker would be easier (3 sections to read instead of 6 for date time)

But for other demos yes dateTime is good

Reply:
Moved to `TimePicker` for the 1st two demos